presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixConfig.java

```diff
@@ -83,4 +89,27 @@ public PhoenixConfig setCaseInsensitiveNameMatchingCacheTtl(Duration caseInsensi
         this.caseInsensitiveNameMatchingCacheTtl = caseInsensitiveNameMatchingCacheTtl;
         return this;
     }
+
+    public Properties getConnectionProperties()
```

Comment:
Make this a private method in `PhoenixClientModule`. Config classes should be simple beans.

Reply:
It cannot be private as it is also used by PhoenixClient